cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/test/java/io/cdap/cdap/datapipeline/DataPipelineTest.java

```diff
@@ -625,8 +626,17 @@ private WorkflowManager deployPipelineWithSchedule(String pipelineName, Engine e
 
     String defaultNamespace = NamespaceId.DEFAULT.getNamespace();
     // Use properties from the triggering pipeline as values for runtime argument key1, key2
+    ArgumentMapping key1MappingWithId = new ArgumentMapping(
+        key1Mapping.getSource(),
+        key1Mapping.getTarget(),
+        new TriggeringPipelineId(defaultNamespace, triggeringPipelineName));
+    PluginPropertyMapping key2MappingWithId = new PluginPropertyMapping(
+        key2Mapping.getStageName(),
+        key2Mapping.getSource(),
+        key2Mapping.getTarget(),
+        new TriggeringPipelineId(defaultNamespace, triggeringPipelineName));
```

Comment:
this doesn't really test that the logic is correct. Should have a test where both pipelines have the same property name in order to make sure the triggering pipeline name is taken into account correctly

Reply:
will update with a new dedicated test